resources/views/ItemEdit/ItemEdit.blade.php

```diff
@@ -0,0 +1,304 @@
+
+    <div class="container">
+        <div class="row justify-content-center">
+
+            {{-- to edit item details --}}
+            <form action="#" method="post" enctype="multipart/form-data" id="form1">
+                @csrf
+                @method('PATCH')
+
+            </form>
+
+            <form action="#" method="post" id="form2">
+                @csrf
+                @method('DELETE')
+            </form>
+
+                <h1 class="text-center">EDIT ITEMS</h1>
+
+
+                <div class="row">
+                    {{-- Item card --}}
+                    <div class="col-4">
+                        <div class="row">
+                            <div class="col-10 align-items-center">
+                                <div class="card card-potision">
+                                    <div class="card-body">
+                                        <div class="upload-box">
+                                            <input type="file" name="upload_file" id="input_files" >
+                                            <span>+</span>
+                                        </div>                                
+                                    </div>
+                                    <div class="card-footer" style="background-color: rgb(216,230,233)">
+                                        <div class="border border-rounded" style="background-color: rgb(153,204,255)">
+                                            <p class="h2 text-center">ITEM NAME</p>
+                                            <p class="h4 text-center">Chicken</p>
+                                            <p class="h5 px-1">PRICE :</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-10">
+                                <p class="text-margin">DRAG-AND-DROP TO CHANGE IMAGES</p>
+                            </div>
+                        </div>
+                    </div>
+                    {{-- Those inputs edit item information --}}
+                    <div class="col-4">
+                        <label for="item-name" class="form-label"></label>
+                        <input type="name" id="item_name" name="item-name" class="form-control text-center input" placeholder="Change Item Name" form="form1">
+
+                        <label for="price" class="form-label"></label>
+                        <input type="price" step="0.01" name="price" id="price" class="form-control text-center input" placeholder="Change Price" form="form1">
+
+                        <label for="category" class="form-label"></label>
+                        <select name="category" id="category" class="form-select" form="form1">
+                            <option value="choose Category">Choose Category</option>
```

Comment:
Maybe you can add this to database so that it is more maintainable

Reply:
I understand. but It does'nt make sense how should I remake code.
I'll give it some thought, do you have any reference